wireapp/wire-android-reloaded
# fix: fix bug not displaying asset messages correctly

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

This PR contains the fix in kalium needed to show correctly on AR, asset messages sent by web clients 

Needs releases with:

https://github.com/wireapp/kalium/pull/413

#### How to Test

Log in on your web client account, try to send an asset message to a given conversation. Open the AR app, check that the asset placeholder gets displayed correctly on the same conversation

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
